helm/goldilocks-app/values.yaml

```diff
@@ -14,11 +14,10 @@ metrics-server:
     create: true
 
 image:
-  registry: quay.io
```

Comment:
Did we lose the possibility to configure registry this way?
This is a standard value that is overridden for Chinese installations, if we don't have it we will need some special config for this app.

Reply:
well they have changed from registry to repository. So the solution is to revert this change manually? I am not used to get upstream changes

Reply:
should I include it then in the repo name? WDYT?